parse_name_from_email.gemspec

```diff
@@ -19,7 +19,13 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'activesupport'
+  if RUBY_VERSION >= '2.3'
+    spec.add_dependency 'activesupport'
+  elsif RUBY_VERSION >= '2.1'
+    spec.add_dependency 'activesupport', '~> 4.1'
+  elsif RUBY_VERSION < '2.1'
+    spec.add_dependency 'activesupport', '~> 3.2'
+  end
```

Comment:
I don't have experiences with defining dependencies for Travis CI, could you help me pls? :)